pylsp/plugins/_rope_task_handle.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import logging
+from typing import List, Optional, Sequence, Callable, ContextManager
+from rope.base.taskhandle import BaseJobSet, BaseTaskHandle
```

Comment:
```suggestion

from rope.base.taskhandle import BaseJobSet, BaseTaskHandle
```


Rope is a third-party plugin, so this import needs to go in its own line.